File: namsor/typescript-node/src/ChineseNameCandidatesExample.ts
import * as fs from 'fs';
import api from "namsor_client"
import models from "namsor_client"

const apiCaller = new api.ChineseApi();
apiCaller.setApiKey(api.ChineseApiApiKeys.api_key, "YOUR_API_KEY");

apiCaller.chineseNameCandidates(
  "Hong", // chineseSurnameLatin
  "Yu", // chineseGivenNameLatin
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling ChineseApi#chineseNameCandidates:");
  console.log(error.body);
});

File: namsor/kotlin/src/main/kotlin/oseg/namsor_examples/ChineseNameCandidatesExample.kt
package oseg.namsor_examples

import app.namsor.client.infrastructure.*
import app.namsor.client.apis.*
import app.namsor.client.models.*

import java.io.File
import java.time.LocalDate
import java.time.OffsetDateTime
import kotlin.collections.ArrayList
import kotlin.collections.List
import kotlin.collections.Map

class ChineseNameCandidatesExample
{
    fun chineseNameCandidates()
    {
        ApiClient.apiKey["api_key"] = "YOUR_API_KEY"

        try
        {
            val response = ChineseApi().chineseNameCandidates(
                chineseSurnameLatin = "Hong",
                chineseGivenNameLatin = "Yu",
            )

            println(response)
        } catch (e: ClientException) {
            println("4xx response calling ChineseApi#chineseNameCandidates")
            e.printStackTrace()
        } catch (e: ServerException) {
            println("5xx response calling ChineseApi#chineseNameCandidates")
            e.printStackTrace()
        }
    }
}
